cmd/snap/cmd_advise.go

```diff
@@ -205,11 +210,50 @@ func adviseViaAptHook() error {
 	return nil
 }
 
+type Snap struct {
+	Snap string
+	Version string
+	Command string
+}
+
+func dumpDbHook() error {
+	commands, err := advisor.DumpCommands()
+	if err != nil {
+		return err
+	}
+
+	commands_processed := make([]string, 0, len(commands))
+	var b[] Snap;
+
+	for key, value := range commands {
+		err := json.Unmarshal([]byte(value), &b)
+		if err != nil {
+			return err
+		}
+		for i := range b {
+			var s = fmt.Sprintf("%s\xff%s/s\n", key, b[i].Snap)
+			commands_processed = append(commands_processed, s)
+		}
+	}
+
+	sort.Strings(commands_processed)
+
+	for _, value := range commands_processed {
+		fmt.Printf("%s", value)
```

Comment:
you could just `fmt.Print(value)`, but really you want `fmt.Fprint(Stdout, value)` (tests replace `Stdout`)